ignite/pkg/cosmoscmd/root.go

```diff
@@ -170,7 +170,7 @@ func NewRootCmd(
 				return err
 			}
 
-			startProxyForTunneledPeers(initClientCtx, cmd)
+			// startProxyForTunneledPeers(initClientCtx, cmd)
```

Comment:
Why is this call commented instead of removed?

Reply:
Because it's that old `cosmoscmd` that should be removed.

Reply:
Note `cosmoscmd` is also marked as deprecated.